src/test/java/uk/gov/hmcts/ecm/common/model/servicebus/tasks/UpdateDataModelBuilder.java

```diff
@@ -11,7 +13,9 @@ public UpdateDataModelBuilder withJurisdictionCode(String jurCode, String outcom
         jurCodeType.setJuridictionCodesList(jurCode);
         jurCodeType.setJudgmentOutcome(outcome);
 
-        dataModel.setJurCodesType(jurCodeType);
+        ArrayList<JurCodesType> list = new ArrayList<>();
+        list.add(jurCodeType);
+        dataModel.setJurCodesList(list);
```

Comment:
I feel like this could maybe be something more concise like dataModel.setJurCodesList(List.of(jurCodeType));

But hard to tell without full context

Reply:
We can't use List.of because it creates an immutable list, which isn't very useful when tests want to add more to it